styles.css

```diff
@@ -78,15 +82,103 @@ h2 {
   color: #f0e3ca;
 }
 
-.h3-container {
+h3 {
+  font-weight: 700;
+  font-size: 2rem;
+  text-align: center;
+  color: #ff8303;
+}
+
+.p-container {
   width: 18em;
   margin-left: auto;
   margin-right: auto;
 }
 
-h3 {
+p {
   font-weight: 400;
   font-size: 1.5rem;
   text-align: justify;
   color: #f0e3ca;
+  background-color: transparent;
+}
+
+hr {
+  color: #111;
+}
+
+/* This code is for the logo at the beging of the About Me section */
+#aboutMeLogo {
+  margin: 0 auto;
+  height: 4em;
+}
+
+.skillsContainer {
+  display: grid;
+  justify-content: center;
+}
+
+.skillsContainer > div {
+  display: inline-flex;
+  margin: 1em;
+  padding-left: 5em;
+  height: 3.75em;
+  width: 17.8em;
+  background-color: #21201d;
+  border-radius: 0.75em;
+  gap: 1em;
+  align-items: center;
+}
+
+.myWorks {
+  display: grid;
+  margin-left: auto;
+  margin-right: auto;
+  width: 24em;
```

Comment:
Nice job! 👍 I tested it and saw it was great! Only suggest reviewing this width and taking a look at the view responsive < 350px

Reply:
Thank you for your review @danyhoshi 
Changes have been applied and tested with screen size as low as 250px.
It works good so far.
I also added my avatar and the header bar menu is always available on top.
I tried to use dvw unit for responsive design, but it triggered some errors when checking linters, so as you suggested I changed the unit to vw.